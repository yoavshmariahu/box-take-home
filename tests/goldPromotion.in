k a1
K e5
g b1
G d2

[S B R P]
[p r b s]

move b1 a2
move d2 d1 promote
