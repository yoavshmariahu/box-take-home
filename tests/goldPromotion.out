UPPER player action: move d2 d1 promote
5 |__|__|__|__| K|
4 |__|__|__|__|__|
3 |__|__|__|__|__|
2 | g|__|__| G|__|
1 | k|__|__|__|__|
    a  b  c  d  e

Captures UPPER: S B R P
Captures lower: p r b s

lower player wins.  Illegal move.
